src/convex/ConvexAvatarMultiToken.sol

```diff
@@ -119,10 +107,8 @@ contract ConvexAvatarMultiToken is BaseAvatar, ConvexAvatarUtils, PausableUpgrad
 
     event ClaimFrequencyUpdated(uint256 newClaimFrequency, uint256 oldClaimFrequency);
 
-    event MinOutBpsCrvToWethValUpdated(uint256 newValue, uint256 oldValue);
-    event MinOutBpsCvxToWethValUpdated(uint256 newValue, uint256 oldValue);
-    event MinOutBpsFxsToFraxValUpdated(uint256 newValue, uint256 oldValue);
-    event MinOutBpsWethToDaiValUpdated(uint256 newValue, uint256 oldValue);
+    event MinOutBpsValUpdated(address tokenA, address tokenB, uint256 newValue, uint256 oldValue);
+    event MinOutBpsMinUpdated(address tokenA, address tokenB, uint256 oldValue, uint256 newValue);
```

Comment:
why `newValue, oldValue` in one and `oldValue, newValue` in the other?

Reply:
looks like you are using it as new,old in L256

Reply:
good catch